analytics/views/stats.py

```diff
@@ -55,11 +55,18 @@ def render_stats(
     analytics_ready: bool = True,
 ) -> HttpResponse:
     assert request.user.is_authenticated
+
+    # Same query to get guest user count as in get_seat_count in corporate/lib/stripe.py.
+    guest_users = UserProfile.objects.filter(
+        realm=request.user.realm, is_active=True, is_bot=False, role=UserProfile.ROLE_GUEST
+    ).count()
```

Comment:
I added a paired comment with this in `corporate/lib/stripe.py`. It's possible a better plan would be to refactor this to a small function called from both places; but my experience is only putting a "same query" comment in one place often leads to divergence.